frontend/src/components/auth/VerificationEmail.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Container, IconButton, Text } from "@chakra-ui/react";
```

Comment:
should `VerificationEmail.tsx` and `reducer.ts` go inside the signup folder?

Reply:
yeah I can move it, should we move the auth reducer to the reducer folder itself or keep here?

Reply:
yup can u do that?